eXo/Sources/Controllers/HomePage/HomePageViewController.swift

```diff
@@ -162,17 +167,14 @@ class HomePageViewController: eXoWebBaseController, WKNavigationDelegate, WKUIDe
             self.navigationController?.popViewController(animated: true)
             UserDefaults.standard.setValue(false, forKey: "wasConnectedBefore")
             UserDefaults.standard.setValue("", forKey: "serverURL")
+            UserDefaults.standard.setValue(false, forKey: "isLoggedIn")
         }
         let serverDomain = URL(string: self.serverURL!)?.host
-        // Page Login Address: [Domain]/portal/login
-        // Page Register: [Domain]/portal/intranet/register
-        //(show navigation bar when the webview display this pages, because the pages don't contain a embedded navigation bar.
-        if (request.url?.absoluteString.range(of: serverDomain! + "/portal/login") != nil) || (request.url?.absoluteString.range(of: serverDomain! + "/portal/intranet/register") != nil) || (request.url?.absoluteString.range(of: serverDomain! + "/rest/state/status") != nil) {
-            self.navigationController?.setNavigationBarHidden(false, animated:true)
-        }else{
-            // Display the navigation bar at login or register page && disable the bar when login (register) is finished
-            // Home Page Address: portal/intranet/register (hide the navigation bar)
-            self.navigationController?.setNavigationBarHidden(true, animated:true)
+        // Display the navigation bar at login or register page && disable the bar when login (register) is finished
+        // Home Page Address: portal/intranet/register (hide the navigation bar)
```

Comment:
```suggestion
        // Display the navigation bar at login or other pages accessible for anonymous users && display the bar when luser is logged in
        // Home Page Address: portal/dw 
```

Reply:
Yes , is better and clear enough like that

Reply:
@wbenabdessalem could you commit this change ?

Reply:
D'accord

Reply:
Simply click here ;) 
![image](https://user-images.githubusercontent.com/27370604/120791993-cccaae80-c52c-11eb-9610-d8e1ff05f64f.png)